License fix

Jyyjy opened the pull request:
Removed duplicated documentation from source code so RAT could run. Added AL 2.0 to all packaged files.

EandrewJones commented:
Did you rebase? There shouldn't be conflicts since you deleted these files, so the diverging history in your branch must need to be reconciled with dev.

Otherwise it looks fine.

Jyyjy committed:
docs(distill): removed docs and added licenses

Removed duplicated documentation from source code so RAT could run. Added AL 2.0 to all packaged files.

Jyyjy commented:
> Did you rebase? There shouldn't be conflicts since you deleted these files, so the diverging history in your branch must need to be reconciled with dev.
> 
> Otherwise it looks fine.

Rebased
